distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: New cheats: No money, add/set custom money, ignore money, freebuilder mode, invent all, own all land. 
```

Comment:
I presume not all things the cheat does have to be mentioned more than the titles/topics of what it does,
for example all three 'No money, add/set custom money, ignore money' could be maybe summed up